core/state_transition.go

```diff
@@ -217,40 +221,52 @@ func (st *StateTransition) TransitionDb() (ret []byte, usedGas uint64, failed bo
 
 	var (
 		evm = st.evm
-		// vm errors do not effect consensus and are therefor
+		// vm errors do not effect consensus and are therefore
 		// not assigned to err, except for insufficient balance
 		// error.
 		vmerr error
 	)
-	log.Debug("Applying new transaction (technically Message)!", "Tx (aka Message) data", st.msg)
+	log.Debug("Applying transaction", "tx data", msg)
 	executionMgrTime := st.evm.Time
 	if executionMgrTime.Cmp(big.NewInt(0)) == 0 {
 		executionMgrTime = big.NewInt(1)
 	}
+
+	// TODO: queue origin is always 0 with current version of em
```

Comment:
Will debug the gas issues with `queueOrigin == 2` (Sequencer) when updating the execution manager to the latest contracts.